src/operator/contrib/open_lstm_rnn.cc

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * Copyright (c) 2018 by Contributors
+ * \file open_lstm_rnn.cc
+ * \brief LSTM RNN Open-Source CUDA Implementation
+ * \author Bojian (Jack) Zheng, Gennady Pekhimenko
+ */
+#include "./open_lstm_rnn-inl.h"
+
+namespace mxnet {
+namespace op {
+
+template <>
+Operator *CreateOp<cpu>(OpenLSTMRNNParam param, int dtype) {
+  LOG(FATAL) << "OpenLSTMRNN is only available for gpu at the moment.";
+
+  Operator * op = nullptr;
+
+  MSHADOW_REAL_TYPE_SWITCH(dtype, DType,
```

Comment:
why is this needed?

Reply:
@apeforest This was written analogously with the [`cuDNN` implementation of LSTM RNN](https://github.com/apache/incubator-mxnet/blob/v0.12.0/src/operator/rnn.cc).

Reply:
@apeforest This was written analogously with the [`cuDNN` implementation of LSTM RNN](https://github.com/apache/incubator-mxnet/blob/v0.12.0/src/operator/rnn.cc).